editors/code/package.json

```diff
@@ -1,31 +1,33 @@
 {
     "name": "clarity-lsp",
-    "displayName": "clarity-lsp",
-    "description": "Language Server Protocol implementation for Clarity",
-    "preview": true,
+    "displayName": "Clarity LSP",
+    "description": "Language Server Protocol implementation for Blockstack's Clarity smart contracting language",
+    "publisher": "blockstack",
     "private": true,
     "version": "0.1.0",
     "releaseTag": null,
-    "publisher": "lgalabru",
-    "repository": {
-        "url": "https://github.com/lgalabru/clarity-lsp.git",
-        "type": "git"
-    },
-    "homepage": "https://github.com/lgalabru/clarity-lsp.git",
+    "repository": "https://github.com/lgalabru/clarity-lsp.git",
+    "homepage": "https://github.com/lgalabru/clarity-lsp",
+    "bugs": "https://github.com/lgalabru/clarity-lsp/issues",
     "license": "GPL-3.0-only",
     "keywords": [
-        "clarity",
-        "blockstack",
-        "blockchain",
-        "smart-contract",
-        "lsp",
-        "language-server"
+        "Blockstack",
+        "Stacks 2.0",
+        "Smart Contract",
+        "Blockchain",
+        "LISP"
```

Comment:
What's the rationale for replacing `Clarity` with `LISP`?
I think we should keep `Language server` / `LSP`.

Reply:
we could discuss it. I simply copied the same tags from the other extension we have, so they show up together for similar searches. My thinking was simple:
- language server protocol is already in the description
- lsp is in the name itself

Hence, repeating the same word won't yield more discoverability. So, instead, I thought we'd add another new word.

Happy to abandon this and stick to LSP instead of lisp